helm_deploy/values-dev.yaml

```diff
@@ -32,3 +32,12 @@ resources:
   memory:
     limit: 1200Mi
     request: 350Mi
+
+generic-prometheus-alerts:
+  targetApplication: court-case-matcher
+  alertSeverity: slack-hmpps-user-preferences
```

Comment:
Will send messages to `probation_in_court_alerts`
I wanted to get something in place ASAP - would be better to go with setting up webhooks for dev/preprod/prod but that would be much more work

Reply:
Hi, we have webhooks setup already  to go to the specific slack channels

1. severity = "probation_in_court_alerts_dev"
2. severity = "probation_in_court_alerts_preprod"
3. severity = "probation_in_court_alerts_prod"

Reply:
Hi, we have webhooks setup already  to go to the specific slack channels

1. severity = "probation_in_court_alerts_dev"
2. severity = "probation_in_court_alerts_preprod"
3. severity = "probation_in_court_alerts_prod"